### DISCUSSION TITLE
Organizando a Estrutura do Curso

### DISCUSSION THREAD

**Discussion by cmsouzaac** [balta-io/2801#88] [Perguntas e Respostas] [answered]
Professor Andre,

Quero para cada aula organizar um arquivo por aula e realizar a execução, porem não consigo executar o segundo arquivo e diante. Como faço pra realizar esse procedimento.

exemplo;
![image](https://user-images.githubusercontent.com/48097255/176001607-b676d140-0d6e-48f4-a0c0-b5c0790b2777.png)

ou vamos aprender esse organização de arquivos na mesma pasta mais adiante?

**Comment by andrebaltieri** [comment]
Bom dia @cmsoouza, qual erro está acontecendo?
Em relação a organizar os arquivos, vamos aprender melhor nas sequências :)

**Comment by cmsouzaac** [comment, marked as answer]
Professor, 
tenho dois arquivos; comentarios/program. 

![image](https://user-images.githubusercontent.com/48097255/176342261-13367413-e93f-4b3f-81a0-961251eeab7d.png)


Percebi que  logo quando estava iniciando a execução no arquivo comentarios, o terminal me retornava a seguinte mensagem de erro;

```
error CS0017: Programa tem ma
is de um ponto de entrada definido. Compilar com /main para 
especificar o tipo que contém o ponto de entrada. [E:\MeuApp
\MeuApp.csproj]
```

entao entendi que tenho que renomear o segundo metodo main para main2, assim quando rodo novamente o "dotnet run" a execução do primeiro arquivo subiu. Porem quando tento executar o segundo arquivo (comentario) o terminal so me retorna as informações do primeiro arquivo (program). 

Como faço pra executar o segundo arquivo?

**Reply by andrebaltieri** [reply]
Entendi... no caso não tem como ter dois programas (Pontos iniciais) no mesmo projeto... eles teriam que ser projetos separados.
Porém você pode criar classes com nomes diferentes, como verá no curso de orientação a objetos...

**Reply by cmsouzaac** [reply]
Blz professor, é pq sou muito ansioso. Vou dar prosseguimento o curso! abs

**Reply by andrebaltieri** [reply]
hahahahaha... vai com calma que dá tudo certo <3
